src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CreateSDORespondent1NotificationHandler.java

```diff
@@ -55,34 +58,23 @@ public List<CaseEvent> handledEvents() {
     private CallbackResponse notifyRespondentSolicitor1SDOTriggered(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
 
-        notificationService.sendMail(
-            caseData.getRespondentSolicitor1EmailAddress(),
-            caseData.getCaseAccessCategory() == CaseCategory.SPEC_CLAIM
-                ? notificationsProperties.getSdoOrderedSpec()
-                : notificationsProperties.getSdoOrdered(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
-        );
+        if (NOTIFY_RESPONDENT_SOLICITOR1_SDO_TRIGGERED.name().equals(callbackParams.getRequest().getEventId())) {
+            if (caseData.getSpecRespondent1Represented() == YesOrNo.YES
+                || caseData.getRespondent1Represented() == YesOrNo.YES) {
+                lr1NotificationSender.notifyRespondentPartySDOTriggered(caseData);
+            } else {
+                lip1NotificationSender.notifyRespondentPartySDOTriggered(caseData);
+            }
+        } else if (caseData.getRespondent2() != null) {
```

Comment:
also if we have both respondent1 and respondent2 then respondent2 statement will not work?

Reply:
okie